spec-0004/index.md

```diff
@@ -102,6 +102,18 @@ workflow" add the following lines as an additional step:
 
 Complete examples of how projects implement this in their CI setup are linked in the Notes section.
 
+#### Artefact cleanup-policy at https://anaconda.org/scientific-python-nightly-wheels
+
+To avoid hosting outdated development versions, as well as to clean up space, we do have a
+default retention policy of:
+
+- Latest **5 versions**
+- Artefacts newer than **30 days**
```

Comment:
```suggestion
#### Artifact cleanup-policy at https://anaconda.org/scientific-python-nightly-wheels

To avoid hosting outdated development versions, as well as to clean up space, we do have a
default retention policy of:

- Latest **5 versions**
- Artifacts newer than **30 days**
```

I think we should stick with the American spelling, but don' t feel strongly about this if you prefer keeping the British spelling.

Reply:
I like the British spelling, but it's not my hill 😉

Reply:
Ah, but you didn't change the PR title, and this repo uses squash merge which defaults to that, so you did manage to sneak one in somewhere. 😉

Reply:
Oh, that wasn't intentional, but from now on I know the trick of how to put back all the 'u'-s and 's'-es where they belong 😉